AuthenticatorPro/Source/Data/Authenticator.cs

```diff
@@ -314,7 +316,7 @@ public string GetOtpAuthUri()
 
         public static string CleanSecret(string input, AuthenticatorType type)
         {
-            if(type == AuthenticatorType.Totp || type == AuthenticatorType.Hotp)
+            if(type == AuthenticatorType.Totp || type == AuthenticatorType.Hotp || type == AuthenticatorType.SteamOtp)
```

Comment:
Replace check with type.IsHmacBased() perhaps. Should be cleaner.

Reply:
As with one of the other comments, probably better to move this into the `AuthenticatorType` directly at some point.